target/scripts/startup/setup-stack.sh

```diff
@@ -640,9 +612,15 @@ function _setup_dkim_dmarc
 
     # shellcheck disable=SC2016
     sed -i -E 's|^(smtpd_milters =.*)|\1 \$dmarc_milter|g' /etc/postfix/main.cf
+  else
+    sed -i '/.*inet:localhost:8891/d' /etc/postfix/main.cf
   fi
 
-  [[ ${ENABLE_OPENDKIM} -eq 1 ]] || return 0
+  if [[ ${ENABLE_OPENDKIM} -eq 0 ]]
+  then
+    sed -i '/.*inet:localhost:8893/d' /etc/postfix/main.cf
+    return 0
+  fi
```

Comment:
Same here, but note you're currently checking if OpenDKIM is disabled, while removing the `dmarc_milter` setting?

Reply:
Done in cef100363fd6765954a7dd0af3ed76b3da8b4d01. Took a slightly different approach now: commented the milter and uncomment them when DKIM/DMARC is enabled.